Slim/Player/Protocols/HTTP.pm

```diff
@@ -732,7 +732,7 @@ sub requestString {
 		$first ||= $song->track->audio_offset if $song->stripHeader || defined $song->initialAudioBlock;
 		
 		if ($first) {
-			$request .= $CRLF . 'Range: bytes=' . ($first || 0) . '-' . ($song->track->audio_size || '');
+			$request .= $CRLF . 'Range: bytes=' . ($first || 0) . '-';
```

Comment:
Is it ok to only define the first byte, but not end?

And the `($first || 0)` is a bit redundant, as there's the `if ($first)` check first.

Reply:
Ok, it is: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Range

Reply:
Yes, I can change that as well. That the result of a different code structure at the beginning